File: Assets/ShootCubes.js
#pragma strict

function Start () {

/*
Simple shooting on walls

1. Create an empty object (Call it bulletHolder)
2. Create a prefab  (Call it bullet and drag an object that you want it to be: cubes)
3. Attach a rigidbody to bullet to activate the script
4. Attach this shoot script to bulletHolder and drag the prefab to the projectile to the bullet holder
5. Make sure the bulletHolder and MainCamera are dragged under the player 

*/


}
var projectile : Rigidbody;
var speed = 20;
function Update () {

//Fire1 is just a tap on the mouse

if ( Input.GetButton ("Fire1")) {
var clone : Rigidbody;
clone = Instantiate(projectile, transform.position, transform.rotation);
clone.velocity = transform.TransformDirection( Vector3 (0, 0, speed));

Destroy (clone.gameObject, 3);



}

}

File: Assets/debug camera control/ShootCubes.cs
using UnityEngine;
using System.Collections;

public class ShootCubes : MonoBehaviour {
	public Rigidbody projectile;
	public int speed=20;
	public int destruct=3;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
		if(Player.playerSingleton.selectMode)
		{
			if ( Input.GetKey (KeyCode.Mouse0)) {
				
				Rigidbody clone;
				 clone = Instantiate(projectile, transform.position, transform.rotation) as Rigidbody;
				clone.velocity = transform.TransformDirection( new Vector3 (0, 0, speed));
				
				Destroy (clone.gameObject, destruct);
			}
		}	
		else{
			if(	Input.GetKeyDown(KeyCode.Mouse0))
			{

					chunk.editCube(InventoryGUI.currentSelection);

					//chunk.editCube(0x80,InventoryGUI.currentSelection.);
					


				//	print("Hit something");
			}
		}
	}
}
